pandas/core/internals/__init__.py

```diff
@@ -2737,7 +2739,7 @@ def _try_coerce_args(self, values, other):
 
     def _try_coerce_result(self, result):
         """ reverse of try_coerce_args """
-        if isinstance(result, np.ndarray):
+        if isinstance(result, (np.ndarray, Block)):
```

Comment:
this is not a good change, please remove. the point is that these are scalars/arrays NEVER blocks.